src/module/entities/helper.js

```diff
@@ -215,7 +215,7 @@ module.exports = class UserProjectsHelper {
 	 * @returns {Array} - List of all sub list entities.
 	 */
 
-	static subEntityList(entities, entityId, type, search, limit, pageNo) {
+	static subEntityList(entities, entityId, type, search, limit, pageNo, language) {
```

Comment:
@MallanagoudaB when ever you add a new param to function, Please update function signature